src/answer/precious.rs

```diff
@@ -235,8 +235,13 @@ fn parse_answer_instruction(
     let code_elements: [&str; 2] = ["pre", "code"];
     for code_element in &code_elements {
         if let Some(title) = answer_node.find(Name(*code_element)).next() {
+            println!("{:?}", title);
```

Comment:
The `println!` message is not necessary, cause it will affect output.  The better way to handle this is using `debug!`

Reply:
Forgot advanced debugging technique stuff 😂